doc-Managing_Infrastructure_and_Inventory/topics/Tagging_Orchestration_Stacks.adoc

```diff
@@ -0,0 +1,11 @@
+= Tagging Orchestration Stacks
+
+Apply tags to orchestration stacks to categorize them together at the same time. You can create tags using the instructions in the https://access.redhat.com/documentation/en/red-hat-cloudforms/4.1/general-configuration/general-configuration[General Configuration] guide.
```

Comment:
@adahms Yeah, good point. And certainly worth collecting all relevant sections in a workflow in the same place where possible at some point.